src_c/surface.c

```diff
@@ -342,6 +344,8 @@ static struct PyMethodDef surface_methods[] = {
      DOC_SURFACEBLIT},
     {"blits", (PyCFunction)surf_blits, METH_VARARGS | METH_KEYWORDS,
      DOC_SURFACEBLITS},
+    {"ublits", (_PyCFunctionFast)surf_ublits, METH_FASTCALL,
```

Comment:
I believe it should be cast to a PyCFunction. Look at `blits` right above. Due to the VARARGS | KEYWORDS, the method is of of type `PyCFunctionWithKeywords`, yet it is cast to a `PyCFunction`.

The docs specify that the blocks in the PyMethodDef must be PyCFunctions. Does this make sense to me: no. But it must work, since that's how everything else does it.

Reply:
Updated it, now only build that fails is the first one for AppVeyor with pypy3.6